ocfweb/docs/markdown_based.py

```diff
@@ -56,5 +64,6 @@ def get_markdown_docs():
         yield Document(
             name='/' + name,
             title=meta['title'],
-            render=partial(render_markdown_doc, meta, text)
+            render=partial(render_markdown_doc, meta, text),
+            path=path,
```

Comment:
That's a good point, since `path` is only needed in the `render_markdown_doc` function. I'll change that.